integreat_cms/core/settings.py

```diff
@@ -80,6 +80,10 @@
 #: Authentication token for the Firebase API. This needs to be set for a correct usage of the messages feature.
 FCM_KEY = os.environ.get("INTEGREAT_CMS_FCM_KEY")
 
+#: Whether push notifications via Firebase are enabled.
+#: This is ``True`` if :attr:`~integreat_cms.core.settings.FCM_KEY` is set, ``False`` otherwise.
+FCM_ENABLED = bool(FCM_KEY)
```

Comment:
I would prefer to provide some dummy FCM key, if DEBUG is activated. Otherwise, we always have to set some env variable to see the whole Push Notification area.